tests/fixtures/mod.rs

```diff
@@ -4,7 +4,9 @@ use assert_fs::prelude::*;
 use port_check::free_local_port;
 use reqwest::Url;
 use rstest::fixture;
+use std::path::Path;
 use std::process::{Child, Command, Stdio};
+use std::sync::LazyLock;
```

Comment:
The use of `LazyLock` does raise the MSRV to 1.80, if you want to run tests. But miniserve doesn't have a MSRV policy so I thought it's probably fine?

If you don't like this, I can always swap in `once_cell::sync::Lazy`.

Reply:
I'm totally cool with you using modern Rust. :)